### PR TITLE
[Doc Improvement][SEO5 metatitle update for same H1][3948822]

### PR DESCRIPTION
**Pull request by v-ramyas5**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8448c966c53259f8f474b2081e0eb57c67057fd3/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 93 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b5109245-e312-4691-be36-6f5244ef88cd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/get-started/build-basic-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8448c966c53259f8f474b2081e0eb57c67057fd3/msteams-platform%2Fget-started%2Fbuild-basic-tab-app.md) | 86 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bbac608f-bfae-4cfb-a981-8da3456cfb50) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/samesite-cookie-update.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8448c966c53259f8f474b2081e0eb57c67057fd3/msteams-platform%2Fresources%2Fsamesite-cookie-update.md) | 86 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a0cc9fe9-7f53-4ed9-baa3-70e01e25c8f2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-create-dashboard-widget-graph-api-call-in-Teams-toolkit.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/8448c966c53259f8f474b2081e0eb57c67057fd3/msteams-platform%2Fsbs-create-dashboard-widget-graph-api-call-in-Teams-toolkit.yml) | 79 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/60278396-dd38-4241-b893-d6381d7ebef8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/Build-a-dashboard-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8448c966c53259f8f474b2081e0eb57c67057fd3/msteams-platform%2Ftabs%2Fhow-to%2FBuild-a-dashboard-tab-app.md) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a0c39596-077d-4eb1-a5ff-3ab7d67084c5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8448c966c53259f8f474b2081e0eb57c67057fd3/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fconfiguration-page.md) | 91 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/62615e8e-88cd-408c-9e53-493999b521f5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/User-Specific-Views.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8448c966c53259f8f474b2081e0eb57c67057fd3/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FUser-Specific-Views.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/de4aea2b-db71-4493-992a-c394abdf4ef9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-using.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8448c966c53259f8f474b2081e0eb57c67057fd3/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-using.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1da4c8f4-956c-4319-ad9f-141fa50ea0e1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c2b352c32d222acf13058fb58135b29d3758d60/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 93 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1a39d83a-8e8e-4d4b-858a-a1943dfa5e8b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/get-started/build-basic-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c2b352c32d222acf13058fb58135b29d3758d60/msteams-platform%2Fget-started%2Fbuild-basic-tab-app.md) | 86 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f0306cf9-efda-477c-9485-40054706111f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/samesite-cookie-update.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c2b352c32d222acf13058fb58135b29d3758d60/msteams-platform%2Fresources%2Fsamesite-cookie-update.md) | 86 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a41d0b32-be92-4967-b561-8d669e76b2cb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/Build-a-dashboard-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c2b352c32d222acf13058fb58135b29d3758d60/msteams-platform%2Ftabs%2Fhow-to%2FBuild-a-dashboard-tab-app.md) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3fe84e43-b215-424b-ab8e-e8ac55bf399a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c2b352c32d222acf13058fb58135b29d3758d60/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fconfiguration-page.md) | 91 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9f534115-e101-4677-8d6d-3829a57d0d16) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/User-Specific-Views.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c2b352c32d222acf13058fb58135b29d3758d60/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FUser-Specific-Views.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4290eae7-5a74-4504-9249-e88233d83dca) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-using.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c2b352c32d222acf13058fb58135b29d3758d60/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-using.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/65d992e3-286e-4c29-9f4a-162e1aaa85e4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/841e8ae19f9ec12042a1b282a1c3788bdbe04281/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 93 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8dde28da-bafe-414a-ac87-f4eaba079f20) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/plan-to-monetize.md](https://github.com/MicrosoftDocs/msteams-docs/blob/841e8ae19f9ec12042a1b282a1c3788bdbe04281/msteams-platform%2Fconcepts%2Fdesign%2Fplan-to-monetize.md) | 90 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1e14c83e-5fa4-4953-8d25-a1924aeadd8a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/get-started/build-basic-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/841e8ae19f9ec12042a1b282a1c3788bdbe04281/msteams-platform%2Fget-started%2Fbuild-basic-tab-app.md) | 86 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/40ba4f94-10bd-441a-8407-a3872e6fef2e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/samesite-cookie-update.md](https://github.com/MicrosoftDocs/msteams-docs/blob/841e8ae19f9ec12042a1b282a1c3788bdbe04281/msteams-platform%2Fresources%2Fsamesite-cookie-update.md) | 86 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3f286451-45e1-4a84-8b2c-cd37b6147339) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/Build-a-dashboard-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/841e8ae19f9ec12042a1b282a1c3788bdbe04281/msteams-platform%2Ftabs%2Fhow-to%2FBuild-a-dashboard-tab-app.md) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3b881566-3b5a-4682-99ee-a258355fb927) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md](https://github.com/MicrosoftDocs/msteams-docs/blob/841e8ae19f9ec12042a1b282a1c3788bdbe04281/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Fauth-oauth-provider.md) | 89 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3f46479e-b506-4ad6-b180-82bdd66410ae) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/841e8ae19f9ec12042a1b282a1c3788bdbe04281/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fconfiguration-page.md) | 91 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dc58d844-5dbe-4de9-ad2e-92e51dd5e0de) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/User-Specific-Views.md](https://github.com/MicrosoftDocs/msteams-docs/blob/841e8ae19f9ec12042a1b282a1c3788bdbe04281/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FUser-Specific-Views.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/35f41bc9-ce1b-4608-b3fd-da195277348d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/invoking-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/841e8ae19f9ec12042a1b282a1c3788bdbe04281/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Finvoking-task-modules.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/85ced146-0698-4459-8473-a5abea65f4f7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-using.md](https://github.com/MicrosoftDocs/msteams-docs/blob/841e8ae19f9ec12042a1b282a1c3788bdbe04281/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-using.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bf068c9f-e0d8-4e66-b909-dc02c4759342) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/deep-link-application.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Deep link to an application 
+title: Configure Deep Links
```
Comment (v-ramyas5) [inline msteams-platform/concepts/build-and-test/deep-link-application.md:2]: for Apps

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2da118719bfaa328f3279296ae4a18e1a0a8ba65/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 93 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bf2622c1-7c6e-4cab-bf47-01470d790ca6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/plan-to-monetize.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2da118719bfaa328f3279296ae4a18e1a0a8ba65/msteams-platform%2Fconcepts%2Fdesign%2Fplan-to-monetize.md) | 90 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b36add58-0adb-4efb-909f-52038309a405) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/get-started/build-basic-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2da118719bfaa328f3279296ae4a18e1a0a8ba65/msteams-platform%2Fget-started%2Fbuild-basic-tab-app.md) | 86 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e7dbbc7c-616d-4e73-9a74-111085bb2bbd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/samesite-cookie-update.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2da118719bfaa328f3279296ae4a18e1a0a8ba65/msteams-platform%2Fresources%2Fsamesite-cookie-update.md) | 86 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5caf9f2b-de41-404c-a1e3-4d5abd004fa1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/Build-a-dashboard-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2da118719bfaa328f3279296ae4a18e1a0a8ba65/msteams-platform%2Ftabs%2Fhow-to%2FBuild-a-dashboard-tab-app.md) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cb19590a-42d3-4c6c-8f5f-93e57f5c32cd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2da118719bfaa328f3279296ae4a18e1a0a8ba65/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Fauth-oauth-provider.md) | 89 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/668f600a-3505-4cac-b82f-f2b0a4853599) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2da118719bfaa328f3279296ae4a18e1a0a8ba65/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fconfiguration-page.md) | 91 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e031db4f-f0a8-4fb0-ac72-226c2887a720) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/User-Specific-Views.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2da118719bfaa328f3279296ae4a18e1a0a8ba65/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FUser-Specific-Views.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4acfe538-97ca-45cc-8021-8ada273ac5cd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/invoking-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2da118719bfaa328f3279296ae4a18e1a0a8ba65/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Finvoking-task-modules.md) | 93 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/089d92ed-d73a-4dc2-8c6e-27db27a7e1d2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-using.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2da118719bfaa328f3279296ae4a18e1a0a8ba65/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-using.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f31e152b-0e04-4552-8065-2dff5d282bfa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-preethah — APPROVED** [review]
Approved.

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2ae03ee01201d5f0eb1d9b35f4efd6b6d86eea90/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 93 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4daebb42-1a0b-498d-afee-9f7fbabeff0d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/plan-to-monetize.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2ae03ee01201d5f0eb1d9b35f4efd6b6d86eea90/msteams-platform%2Fconcepts%2Fdesign%2Fplan-to-monetize.md) | 90 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ad910d85-e822-49b0-9075-265a476667aa) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/get-started/build-basic-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2ae03ee01201d5f0eb1d9b35f4efd6b6d86eea90/msteams-platform%2Fget-started%2Fbuild-basic-tab-app.md) | 86 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f577fb02-23a0-446c-9ec2-38d5db36e60f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/samesite-cookie-update.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2ae03ee01201d5f0eb1d9b35f4efd6b6d86eea90/msteams-platform%2Fresources%2Fsamesite-cookie-update.md) | 86 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6c96ee89-95a4-4067-9b89-08b852d74bc5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/Build-a-dashboard-tab-app.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2ae03ee01201d5f0eb1d9b35f4efd6b6d86eea90/msteams-platform%2Ftabs%2Fhow-to%2FBuild-a-dashboard-tab-app.md) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f0ec9d4c-8711-401d-bfe2-6024f1d89459) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2ae03ee01201d5f0eb1d9b35f4efd6b6d86eea90/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Fauth-oauth-provider.md) | 89 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/22c0572b-50e5-4229-b0ed-d5b09a8e53a3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2ae03ee01201d5f0eb1d9b35f4efd6b6d86eea90/msteams-platform%2Ftabs%2Fhow-to%2Fcreate-tab-pages%2Fconfiguration-page.md) | 91 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/962bd21e-d344-4f2b-8e27-8a9260d3f898) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/User-Specific-Views.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2ae03ee01201d5f0eb1d9b35f4efd6b6d86eea90/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FUser-Specific-Views.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6bf6c36e-3f6c-41de-90c2-8281fd3c6e06) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/invoking-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2ae03ee01201d5f0eb1d9b35f4efd6b6d86eea90/msteams-platform%2Ftask-modules-and-cards%2Ftask-modules%2Finvoking-task-modules.md) | 93 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f0568e98-0134-4143-a47c-12cab0e44f97) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/webhooks-and-connectors/how-to/connectors-using.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2ae03ee01201d5f0eb1d9b35f4efd6b6d86eea90/msteams-platform%2Fwebhooks-and-connectors%2Fhow-to%2Fconnectors-using.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7a0cf38c-1378-48ff-a9a2-2b265144df90) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**